### DISCUSSION TITLE
Replacement Part for Radio Chip Antenna

### DISCUSSION THREAD

**Discussion by kunalwasnik** [bitcraze/discussions#1643] [Q&A] [answered]
Hi , our radio chip antenna of crazy flie 2.1 drone came off. We were looking for spare replacement. Can you tell us the specific part or alternative to that radio chip antenna , and where can we buy it. (Also what are the consequence if its not there , how much range will be affected ? )

![Screenshot from 2024-11-09 16-25-06](https://github.com/user-attachments/assets/9fb0fe53-cb62-4856-8b84-3a31803c2b78)

Thanks

**Comment by tobbeanton** [comment, marked as answer]
You will need the antenna. A ~23mm wire could work as a backup but you can also get the 2450AT18B100 replacement part.
[2450at18b100.pdf](https://github.com/user-attachments/files/17702885/2450at18b100.pdf)

**Reply by kunalwasnik** [reply]
Thank you very much. :)
